CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 ## 1.6.1
 * Add webhooks into travis.yml to notifiy galaxy about new releases.
 
+## 1.6.1
```

Comment:
@speedlight it looks like there is a duplicate entry here, `1.6.1` section already exists.